symfony/client/src/orangehrmPimPlugin/components/EmployeeSubordinates.vue

```diff
@@ -0,0 +1,203 @@
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div>
+    <save-supervisor
+      v-if="showSaveModal"
+      :http="http"
+      :reporting-methods="reportingMethods"
+      @close="onSaveModalClose"
+    ></save-supervisor>
+    <edit-supervisor
+      v-if="showEditModal"
+      :http="http"
+      :reporting-methods="reportingMethods"
+      @close="onEditModalClose"
+    ></edit-supervisor>
+    <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+      <profile-action-header @click="onClickAdd">
+        Assigned Subordinates
+      </profile-action-header>
+    </div>
+    <table-header
+      :selected="checkedItems.length"
+      :total="total"
+      :loading="isLoading"
+      @delete="onClickDeleteSelected"
+    ></table-header>
+    <div class="orangehrm-container">
+      <oxd-card-table
+        :headers="headers"
+        :items="items?.data"
+        :selectable="selectable"
+        :clickable="false"
+        :loading="isLoading"
+        v-model:selected="checkedItems"
+        rowDecorator="oxd-table-decorator-card"
+      />
+    </div>
+    <div v-if="showPaginator" class="orangehrm-bottom-container">
+      <oxd-pagination :length="pages" v-model:current="currentPage" />
+    </div>
+    <delete-confirmation ref="deleteDialog"></delete-confirmation>
+  </div>
+</template>
+
+<script>
+import usePaginate from '@orangehrm/core/util/composable/usePaginate';
+import {APIService} from '@orangehrm/core/util/services/api.service';
+import ProfileActionHeader from '@/orangehrmPimPlugin/components/ProfileActionHeader';
+import DeleteConfirmationDialog from '@orangehrm/components/dialogs/DeleteConfirmationDialog';
+import SaveSupervisor from '@/orangehrmPimPlugin/components/SaveSupervisor';
+import EditSupervisor from '@/orangehrmPimPlugin/components/EditSupervisor';
+
+export default {
+  name: 'employee-subordinates',
+
+  components: {
+    'profile-action-header': ProfileActionHeader,
+    'save-supervisor': SaveSupervisor,
+    'edit-supervisor': EditSupervisor,
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+
+  props: {
+    employeeId: {
+      type: String,
+      required: true,
+    },
+    reportingMethods: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  setup(props) {
+    console.log('Setup Called')
```

Comment:
pls remove the console.log

Reply:
done. updated the method with correct implementation